app/views/assessments/_handin_form.html.erb

```diff
@@ -69,7 +69,17 @@
             <% end %>
             
             <%= render partial: "submission_panel", locals: {repos: @repos, f: f}%>
-
+            <% filename_list = @assessment.handin_filename.split(".") %>
+            <% if (filename_list.size == 2) %>
```

Comment:
Change check to `> 1`, otherwise it wouldn't render for e.g. `handin.tar.gz`

Reply:
done